packages/fx-core/src/plugins/resource/function/plugin.ts

```diff
@@ -522,7 +530,7 @@ export class FunctionPluginImpl {
       this.config.subscriptionId,
       FunctionConfigKey.subscriptionId
     );
-    const functionAppName = this.checkAndGet(
+    let functionAppName = this.checkAndGet(
```

Comment:
If this value is used only when not arm supported, what about move it to the `if (!isArmSupported)` block?

Reply:
The value will be updated by arm output in arm support scenario.